xds/internal/test/xds_server_serving_mode_test.go

```diff
@@ -261,11 +243,11 @@ func (s) TestServerSideXDS_ServingModeChanges(t *testing.T) {
 	}
 
 	// The clientConns created earlier should be able to make RPCs now.
-	if _, err := client1.EmptyCall(ctx, &testpb.Empty{}, grpc.WaitForReady(true)); err != nil {
-		t.Fatalf("rpc EmptyCall() failed: %v", err)
+	if err := waitForSuccessfulRPC(ctx, cc1); err != nil {
+		t.Fatal(err)
 	}
-	if _, err := client2.EmptyCall(ctx, &testpb.Empty{}, grpc.WaitForReady(true)); err != nil {
-		t.Fatalf("rpc EmptyCall() failed: %v", err)
+	if err := waitForSuccessfulRPC(ctx, cc2); err != nil {
+		t.Fatal(err)
 	}
```

Comment:
If this is literally always a `t.Fatal`, maybe pass `t` there, `t.Helper` it, and save 2 LoC every time it's called?  (Same with `waitForFailedRPC`.)

Reply:
Done.